drivers/clock/meson/timer.c

```diff
@@ -168,7 +169,7 @@ protected(microkit_channel ch, microkit_msginfo msginfo)
             }
             break;
         default:
-            microkit_dbg_puts("DRIVER|ERROR: Unknown request to timer from client\n");
+            dprintf("TIMER DRIVER|LOG: Unknown request to timer from client %llu\n", microkit_msginfo_get_label(msginfo));
```

Comment:
Would you mind also printing out what the channel number is as well? Thanks.

Reply:
Done